firebase-vscode/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## NEXT
 
+## 0.13.1
+
+- Updated internal `firebase-tools` dependency to 13.31.2
+- [Fixed] Insecure operations no longer surfaced as errors.
```

Comment:
I think we can omit this from the changelog -- the issue this fixed is behind an experiment flag still, so it wouldn't have affected any developers.